react/features/base/avatar/constants.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+/**
+ * The base URL for gravatar images.
+ */
+export const GRAVATAR_BASE_URL = 'https://www.gravatar.com/avatar/';
```

Comment:
Can you make sure we use this link also here: https://github.com/jitsi/jitsi-meet/blob/5c67e8b4cec35b6132b61bf13ed8cf2107852bfd/react/features/base/participants/functions.js#L37

Reply:
good catch

Reply:
Done!